### PR TITLE
Create parental leave page

### PR DESCRIPTION
**Pull request by mercadon**

### CODE REVIEW

File: content/benefits-pay-perks/benefits-perks/parental-leave.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### Notify the People team
+
+Notify People Ops via email (people-ops@sourcegraph.com) as soon as you can (no later than 1 month prior to your leave date) with answers to the following questions:
+What date are you planning on starting your leave?
+What is the due date?
+Are you the birthing caregiver or non-birthing caregiver?
+Which country and state will you be based in during your leave?
+Do you plan on taking 16 weeks (or more/less)?
+Will you be taking your leave consecutively or non-consecutively?
+
```
Comment (cassiemelani) [inline content/benefits-pay-perks/benefits-perks/parental-leave.md:20]: -

Comment (cassiemelani) [inline content/benefits-pay-perks/benefits-perks/parental-leave.md:23]: @mercadon What are your thoughts on adding a question: "Are there any questions or concerns you have for the People Ops team regarding your leave?" or something like "Is there any additional support or help you'd like from the People Team?"

Trying to think through a question to pre-emptively answer any questions they might have while making them feel supported.

**Review by cassiemelani — COMMENTED** [review]
https://github.com/sourcegraph/handbook/pull/2890/files#r839016254

File: content/benefits-pay-perks/benefits-perks/parental-leave.md (hunk 1 above)
Comment (mercadon) [reply content/benefits-pay-perks/benefits-perks/parental-leave.md:23]: Let's do it! It'll be good data for us too as we build this out more.

**Pull request merged**